scripts/generateDai.js

```diff
@@ -0,0 +1,45 @@
+// usage:
+// > node_modules/.bin/babel-node scripts/fundTestAccount.js <address>
+
+import Maker from '../packages/dai/src';
```

Comment:
please use absolute imports, i.e. `@makerdao/dai`; once you run the setup ([docs](https://github.com/makerdao/dai.js#getting-started)) they will use the local builds anyway

Reply:
done